lang/cs/Org.Apache.REEF.Bridge.CLR/Message/BridgeProtocol.cs

```diff
@@ -48,17 +48,17 @@ public static string Schema
         public int offset { get; set; }
```

Comment:
Add comments for what is offset?

Reply:
That's a file auto-generated from Avro, can't change it.

Reply:
We want to be able to negotiate the protocol in the future. For example, use integers for messages instead of strings in the header which would result in faster lookups. However, we will look at that once this is all working and this is an artifact from early experiments that can be removed by editing it out of the avsc file.

Reply:
Can you put some comments to describe what it is? Or any future work so that we will remember?